smoke-test/src/test/java/org/opennms/smoketest/minion/DetectorsCommandIT.java

```diff
@@ -178,13 +178,11 @@ public void canLoadDetectorsOnOpenNMS() throws Exception {
             LOG.info("Found detectors: {}",  detectorMap);
 
             // Verify
-            for (String detectorName : expectedDetectors.keySet()) {
-                if (!detectorMap.containsKey(detectorName)) {
-                    unmatchedDetectors.add(detectorName);
-                }
-            }
+            return expectedDetectors.keySet().stream().filter(e -> {
+                // If no line matches the expected value, return it 
+                return detectorMap.keySet().stream().noneMatch(s -> s.contains(e));
```

Comment:
Using contains isn't quite reliable since: `"HTTPS".contains("HTTP") == true`

I'm guessing there's extra white-space in the output?

Reply:
The output has ANSI color codes provided by JLine.